atat_provisioning_wizard_api.yaml

```diff
@@ -977,7 +983,9 @@ components:
       additionalProperties: false
       properties:
         task_order_number:
+          description: TO numbers are 13 characters. TO modifications are 17 characters.
           type: string
+          pattern: '^[\d]{13,17}$'
```

Comment:
Not sure if the full business rule is in scope for this ticket, but this pattern would probably need to modified in the future to meet the full business rule for `Task Order Number`.

Reply:
@juliusfitzhugh-ccpo these were changed to be more specific just a few days ago. We still haven't had the chance to update the ui with these new requirements.  Imo this is sufficient for now, and we can create another ticket to capture the additional business rules